decko-rails/spec/controllers/card_controller_spec.rb

```diff
@@ -4,8 +4,13 @@
 
   include Capybara::DSL
   describe "- route generation" do
+    routes { Decko::Engine.routes }
 
     it "should recognize type" do
+      all_routes = Rails.application.routes.routes
+
+   require 'rails/application/route_inspector'
```

Comment:
Magic so tests know about engine routes.  Probably why the Decko::Engine declaration isn't in the card-gem branch.  Breaks tests without this in the tests.